public/angular/bower.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "fc-app",
+  "description": "",
+  "main": "",
+  "authors": [
+    "<none>"
+  ],
+  "license": "MIT",
+  "homepage": "https://github.com/FastingCrusade/website",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
```

Comment:
If this is actual testing, I think it should be committed. One of things I'm going to do is start running CI tests against the back-end now that @trumpsnare is helping. We could run acceptance tests on Codeception or something too.